### PR TITLE
Stac-13362 Fix for hitting max connection limit too early

### PR DESCRIPTION
**Pull request by rb3ckers**
### Step 1: Link to Jira issue
https://stackstate.atlassian.net/browse/STAC-13362

### Step 2: Description of changes
* Upgrade to epbf trace lib that only logs once per message
* Separate configuration for `maxInFlight` for tracer from `maxConnectionsPerMessage` for agent
* Re-introduce batching for connections

### Step 3: Did you add / update tests for your changes in the right area?
- [ ] Unit/Component test		

### Step 4: I'm confident that everything is properly tested:
I got a PO / QA Approval by:
- [ ] Name

### Step 5: Can we ship this feature to production?
- [ ] Yes, I'm proud of my work. Ship it! :ship:

### CODE REVIEW

File: Gopkg.toml [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
 [[constraint]]
   name = "github.com/StackVista/tcptracer-bpf"
-  version = "=v7.0.3"
+  version = "7.0.4"
```
Comment (julianosk) [inline Gopkg.toml:11]: Does it work without `=V`? On `tcptracer-bpf`, the tag has a `v`

File: checks/net.go [hunk 2]
```diff
@@ -322,26 +323,37 @@ func mergeWithHistogram(metricSketch *ddsketch.DDSketch, rtHist *ddsketch.DDSket
...
-		batch.ClusterName = cfg.ClusterName
-	}
+	for len(cxs) > 0 {
+		batchSize := min(cfg.MaxConnectionsPerMessage, len(cxs))
+
+		batch := &model.CollectorConnections{
+			HostName:    cfg.HostName,
+			Connections: cxs[:batchSize],
+			GroupId:     groupID,
+			GroupSize:   groupSize,
```
Comment (julianosk) [inline checks/net.go:336]: Just for my understanding. Until now we are sending only one batch. Now we are adding each batch to the list, then we will be sending one message with multiple batches of connections.
I still couldn't find where we are calling the Run method. That would be from `stackstate-agent`?

**Review by julianosk — APPROVED** [review]
LGTM. Just some questions for my own understanding.

File: Gopkg.toml (hunk 1 above)
Comment (rb3ckers) [reply Gopkg.toml:11]: Mmmm. Good question. Overlooked that. This translates to `^7.0.4`. Not what we want!

File: checks/net.go (hunk 2 above)
Comment (rb3ckers) [reply checks/net.go:336]: We were indeed. But the original code was doing the batching already, the batching was removed a long time ago. I just brought it back (using the nearly identical code from git history).

The Process agent is its own process, so the run method is called, in the end by the process agent itself. But the Go modules that do that are indeed part of the `stackstate-agent` repository. So the process agent depends on some modules from that repo for check running (that is generic agent functionality) but then instantiates the specific checks (process, connections and/or container) in its own main function based on the `checks` configuration.

Comment (julianosk) [reply checks/net.go:336]: I found it now, thanks! `main.go` -> `main_common.runAgent` -> `collector.run` -> `collector.runCheck` and then we get there. I'm still getting used to the implicit implementation of golang. :)

**Pull request merged**